src/main/java/de/techdev/portal/domain/trackr/TrackrConfiguration.java

```diff
@@ -12,6 +11,12 @@
 @Configuration
 class TrackrConfiguration {
 
+    @Value("${trackr.oauth.resourceId}")
+    private String trackrResourceId = "techdev-services";
+
+    @Value("${techdev.portal.clientId}")
+    private String techdevPortalClientId = "techdev-portal";
```

Comment:
See comment on line 15. Do not initialize this, since the value is set by @Value.